### ISSUE TITLE
No label for input boxes during integration setup

### ISSUE THREAD

**Issue by shamasis** [libdyson-wg/ha-dyson#95]
Hello. Thank you for building and maintaining this amazing integration.

I have a weird issue. I used the method of copying files to `custom_components`. When I go to add integration, no matter what method I select, the subsequent screens do not show any text label for the inputs. (See screenshot)

![image](https://github.com/libdyson-wg/ha-dyson/assets/232373/0103e6c5-cb12-4bce-b483-5b2a059e55c0)

I'm on HA 2023.11.3 version with 11.1 OS version. (Latest as of writing this issue.)

Would be really grateful if I could know what's the issue and any patch to fix this.

Meanwhile, I think the setup would work fine if I knew what these inputs were. Can someone tell me what the input boxes represent if I use "setup automatically using MyDyson account" option?

**Comment by dotvezz** [comment]
Something is _very_ wrong here - this shouldn't happen at all and I'm sorry you have to deal with it. I've just posted a new release that I'm hoping helps with some other issues; I don't think it'll be a solution for what you're seeing here, but I can hope. 

If it's still happening to you after you install v1.3.3 of this integration, please follow up here and we'll continue to troubleshoot.

**Comment by dotvezz** [comment]
It's been a while, I'm hoping that means your issue is resolved. But if it's not, please feel free to reply and we can reopen the thread.

**Comment by shamasis** [comment]
@dotvezz, I am so sorry for ghosting this thread. I had been trying to understand this bug for so long that I just missed looking up here. I come from a Node.js background and have been using Homebridge for a while. On that platform, it is super easy for me to dive deep into code and debug, and you can imagine how that overconfidence cost me a lot of time and headaches on HA (which I am very new to).

Ok, so I upgraded to the latest tagged release on the repo. No help. U then tried upgradring to the latest on main branch, that did not help too.

See the screenshot:
![image](https://github.com/libdyson-wg/ha-dyson/assets/232373/e52b295d-c01b-434f-b4c6-c40147123fd8)

With sheer trial and error, I managed to reach this screen for adding the machine post-discovery.
![image](https://github.com/libdyson-wg/ha-dyson/assets/232373/1278f1cd-f591-4b7e-b738-ff1343c61dfe)

No idea what the first input box means. Setting any value in the first input box said "unknown error". Leaving it blank added the device.

So, in short I am up and running, but the setup wizard labels are still not visible. Hopefully I am a super edge-case and you don't have to hunt for a solution.
